src/acom_music_box/tools/waccmToMusicBox.py

```diff
@@ -129,8 +213,7 @@ def readWACCM(waccmMusicaDict, latitude, longitude,
 
         chemSinglePoint = singlePoint[waccmKey]
         if (True):
-            logger.info("{} = object {}".format(waccmKey, chemSinglePoint))
-            logger.info("{} = value {} {}".format(waccmKey, chemSinglePoint.values, chemSinglePoint.units))
+            logger.info("WACCM chemical {} = value {} {}".format(waccmKey, chemSinglePoint.values, chemSinglePoint.units))
```

Comment:
could the `if (True):` be removed if this is useful diagnostic info for users?

Reply:
Yes, let's do exactly that; remove the True block and undent.
As you know, Louisa and Mary are currently checking chemical values and units, so that third line of code will provide useful diagnostics for them. :-)